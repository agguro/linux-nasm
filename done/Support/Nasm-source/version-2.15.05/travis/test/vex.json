[
	{
		"description": "Test VEX2/VEX3/EVEX prefix",
		"id": "vex",
		"format": "bin",
		"source": "vex.asm",
		"option": "-Ox",
		"target": [
			{ "output": "vex.bin" }
		]
	},
	{
		"description": "Test VEX3 prefix error",
		"ref": "vex",
		"option": "-Ox -DERROR -o vex.bin.err",
		"target": [
			{ "stderr": "vex.stderr" }
		],
		"error": "expected"
	}
]
