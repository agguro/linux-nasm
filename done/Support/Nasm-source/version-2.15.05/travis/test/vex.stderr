./travis/test/vex.asm:8: error: specific encoding scheme not available